include/boost/geometry/strategies/geographic/distance_cross_track.hpp

```diff
@@ -105,6 +111,30 @@ public :
         return distance_type(m_spheroid);
     }
 
+    struct azimuth_strategy
+    {
+        typedef azimuth::geographic<FormulaPolicy, Spheroid, CalculationType> type;
+    };
+
+    inline typename azimuth_strategy::type get_azimuth_strategy() const
+    {
+        typedef typename azimuth_strategy::type azimuth_type;
+        return azimuth_type(m_spheroid);
+    }
+
+    struct envelope_segment_strategy
+    {
+        typedef envelope::geographic_segment<FormulaPolicy, Spheroid, CalculationType> type;
+    };
+
+    inline typename envelope_segment_strategy::type get_envelope_segment_strategy() const
```

Comment:
Same story if not needed by the algorithm but needed by upper-level strategy then should be implemented there. Unless there is no Seg-Box strategy and this Pt-Seg strategy is passed into distance algorithm for Seg-Box arguments. But then it'd mean that the algorithm has to be dispatched by CS because cartesian Pt-Seg strategy returns dummy strategies here.